docs/src/content/docs/community/showcase/xensql.md

```
@@ -0,0 +1,28 @@
+---
+title: XenSQL
+description: Local-first SQL workbench with multi-database support and advanced query tooling
+---
+
+![XenSQL Screenshot](../../../../assets/showcase-images/xensql-1.png)
+<table>
+  <tr>
+    <td align="center"><img src="../../../../assets/showcase-images/xensql-2.png?raw=true" width="100%"></td>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)